.gitignore

```diff
@@ -10,3 +10,4 @@ nbactions.xml
 .gradle
 out
 build
+bin/
```

Comment:
Revert or if you really want to add, it should be a separate PR.

Reply:
Revereted